packages/fast-components-styles-msft/src/label/index.ts

```diff
@@ -0,0 +1,20 @@
+import { IDesignSystem } from "../design-system";
+import { ComponentStyles } from "@microsoft/fast-jss-manager";
+import { ILabelClassNameContract } from "@microsoft/fast-components-class-name-contracts-base";
+import { toPx } from "../utilities/units";
+import { typeRamp } from "../utilities/typography";
+
+const styles: ComponentStyles<ILabelClassNameContract, IDesignSystem> = {
+    label: {
+        fontSize: toPx(typeRamp.t8.vp3.fontSize),
```

Comment:
If you want, we have a function that can return a single type instance now.

Reply:
Since we "override" the lineHeight below, I think it is cleaner to leave as is

Reply:
Looks like the t8 VP3 line height is 20 so it's the same. Seems like something residual from way back when.